### PR TITLE
Refactor PlayerListTable

### PR DESCRIPTION
**Pull request by f117qaz123**

### CODE REVIEW

**Comment by f117qaz123** [comment]
[Refactor PlayerListTable](https://trello.com/c/GARe35GP/9-refactor-playerlisttable)

File: client/src/components/Players/Players.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+import PlayerTableRow from "../PlayerTableRow/PlayerTableRow";
+import TableContainer from "@mui/material/TableContainer";
+import { Paper } from "@mui/material";
+import Table from "@mui/material/Table";
+import TableHead from "@mui/material/TableHead";
+import TableRow from "@mui/material/TableRow";
+import TableBody from "@mui/material/TableBody";
+
+const Players = ({playersData}: { playersData: PlayerBriefData[] }) => {
+  const tableHead = tableCategories.map((tableCategories) => {
```
Comment (LianyuMa) [inline client/src/components/Players/Players.tsx]:
```suggestion
  const tableHeadRowCells =
  <>
  <TableCell>Name</TableCell>
  {tableCategories.map(({label}) => <TableCell key={tableCategories.label}  @align="right">{tableCategories.label}</TableCell>)}
  <TableCell align="center">Team</TableCell>
  </>
```
similarly rename tableRowsCells below

Comment (f117qaz123) [reply client/src/components/Players/Players.tsx]: Create a new component called tableHeadRowCells?

Comment (LianyuMa) [reply client/src/components/Players/Players.tsx]: not necessary, inject like this should be good

Comment (f117qaz123) [reply client/src/components/Players/Players.tsx]: image

**Review by LianyuMa — APPROVED** [review]

**Pull request merged**